source/upgrading/upgrade/_sysvm_templates.rst

```diff
@@ -48,9 +48,7 @@ Update System-VM templates
    |            |                                                            |
    |            | Format: VHD                                                |
    |            |                                                            |
-   |            | OS Type: Debian GNU/Linux 7.0 (64-bit) (or the             |
-   |            | highest Debian release number available in the             |
-   |            | dropdown)                                                  |
+   |            | OS Type: Other Linux 64-bit                                |
```

Comment:
it shows as `Other Linux (64-bit)` is the drop down.

Reply:
fixed.